1.0/learn/how-to-document-ballerina-code/index.html

```diff
@@ -274,15 +274,15 @@ <h2 id="writing-ballerina-documentation">Writing Ballerina Documentation</h2>
 </code></pre></div></div>
 
 <blockquote>
-    <p><strong>Tip:</strong> As a best practice, do not add a full stop at the end of the first sentence of the parameter and return type descriptions. Instead, add a fullstop only at the end of a function description. For Example,</p>
+    <p><strong>Tip:</strong> Always, add a full stop at the end of a function description. However, as a best practice, omit the full stop if you have only one sentence in the parameter and return type descriptions. For example,</p>
```

Comment:
Also need to mention here specifically, that if there are multiple sentences for a parameter or return type description, the full stop is required.

Reply:
Thought that it is evident when we mention this special scenario and no need to mention that explicitly.

Reply:
The first line is regarding the function description. The second line talks about parameters and the return type, which is different.

Reply:
Yes. In the second line, we mention about the only special scenario we have WRT the parameters and the return type and that implies that the full stop is required in all other scenarios of them.

Reply:
Anyway, added that. Thanks!

Reply:
Is this updated in all the occurrences? ..

Reply:
Yes, it is.